specification/communication/data-plane/JobRouter/preview/2021-10-20-preview2/communicationservicejobrouter.json

```diff
@@ -0,0 +1,3635 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Communication Routing Service",
+    "description": "Azure Communication Routing Service",
+    "version": "2021-10-20-preview2"
```

Comment:
Typically, we don't append a number to the "-preview", we would just increment the date.

Reply:
Using the -preview<number> is how ACS services have been doing versioning. I see that some other services use the numbering as well, for example: Chat, so it would be nice to keep it.